### PR TITLE
QER's MBR and GBR IEs payload length should be 10Bytes

### PR DESCRIPTION
**Pull request by muthuramanecs03g**
The current implementation MBR and GBR IEs payload length are 8bytes (UL and DL) only. The spec claims those IEs payload length should be 10bytes (5bytes for UL and 5Bytes for DL). 

In this fix, MBR and GBR IEs payload length are set as 10bytes. 

For UL, filled the given uint32 in the payload from 1 to 4. Index 0 is reserved. 
For DL, filled the given uint32 in the payload from 5 to 9. Index 4 is reserved.

### CODE REVIEW

**Comment by wmnsk** [comment]
Thanks!
Since I’m not sure if it’s right change(probably so, as I haven’t tested enough), I’ll have a look at the spec again later. Two points for you before that;
* If I misunderstood the spec and it’s wrongly implemented, the data used in the test are also wrong. Please fix them as well.
* I don’t think defining constants/variables for length/offset is good idea, as it’s used only a few times in a single file and it’s obvious enough with the raw number.

**Comment by muthuramanecs03g** [comment]
I referenced the  section "8.2.8 MBR" and "8.2.9 GBR" in 3GPP TS 29.244 V15.3.0 (2018-09)

Ok, I will fix your pointed things too.

The size of UL and DL is 5bytes respectively, and it can be filled in many ways.  I will fix like below

For UL & DL, take uint64 as a given input, and the first 5bytes of uint64 will be filled in the payload.

If you have any other suggestions then let me know.

**Comment by wmnsk** [comment]
Confirmed, thanks. No other things in my mind, so please fix the two points I wrote above.

**Comment by muthuramanecs03g** [comment]
Please, review the code changes.

File: ie/gbr.go [hunk 1]
```diff
@@ -7,21 +7,23 @@ package ie
...
-
-	binary.BigEndian.PutUint32(i.Payload[0:4], ul)
-	binary.BigEndian.PutUint32(i.Payload[4:8], dl)
-
+func NewGBR(ul, dl uint64) *IE {
+	i := New(GBR, make([]byte, 10))
+	i.Payload[0] = byte(ul >> 32)
+	i.Payload[5] = byte(dl >> 32)
+	binary.BigEndian.PutUint32(i.Payload[1:5], uint32(ul))
+	binary.BigEndian.PutUint32(i.Payload[6:10], uint32(dl))
```
Comment (wmnsk) [inline ie/gbr.go:20]:
```suggestion
	copy(i.Payload[0:5], utils.Uint64To40(ul))
	copy(i.Payload[5:10], utils.Uint64To40(dl))
```

**Review by wmnsk — COMMENTED** [review]
Thanks! Could you use `utils.Uint64To40` as I suggested instead of putting a byte and the rest 4 bytes respectively?

File: ie/gbr.go (hunk 1 above)
Comment (muthuramanecs03g) [reply ie/gbr.go:20]: Thanks for the information.

**Review by wmnsk — APPROVED** [review]
Looks good, thanks!

**Pull request merged**